addons/packages/velero/metadata.yaml

```diff
@@ -9,5 +9,6 @@ spec:
   providerName: VMware
   maintainers:
     - name: Carlisia Thompson
```

Comment:
We should probably remove Carlisia's name as part of this.

Reply:
OK.